mmv1/products/compute/RegionNetworkEndpointGroup.yaml

```
@@ -168,6 +168,20 @@ properties:
     custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.tmpl'
     resource: 'Subnetwork'
     imports: 'selfLink'
+  - !ruby/object:Api::Type::NestedObject
+    name: 'pscData'
```

Just in the last day we have changed our code generator in a way that impacts the contents of YAML files. To meet the new  code generator's expectations you should only need to remove these Ruby-specific annotations from your PR and instead put the type of a field as a property like below.

Currently the acceptance tests on this PR are failing as a consequence of this (looks like any fields missing the type property are generated as String fields by default.

```suggestion
   - name: 'pscData'
    type: String
    type: NestedObject
```

---

Note: the indenting might be incorrect here, please don't just accept the suggestion!

---

Thanks, the branch was created days ago so did not catch the new code generator.

---

No worries, just unlucky timing!